### DISCUSSION TITLE
git clone is getting error

### DISCUSSION THREAD

**Discussion by RATNARAJUBETHAPUDI** [learndevopsonline/learndevopsonline#745] [Q&A]
![image](https://user-images.githubusercontent.com/88640907/235298188-a265794d-a8a6-4f42-a93b-4be2757d00a3.png)
1. when im using git clone it is askig uname and password but not accepting it also tried to use public aceess token and it is saying password authentication has been removed??

**Comment by Manju9876** [comment]
hello @RATNARAJUBETHAPUDI 
as per my knowledge, I would suggest  you to try 

`sudo git clone https://github.com/......`

**Reply by Arafique458** [reply]
Hi @RATNARAJUBETHAPUDI 

I believe you have your repository set as private. This is why you are being asked to enter the credentials.

**Reply by Manju9876** [reply]
hi @RATNARAJUBETHAPUDI 

### hope soo !!!

**Comment by RATNARAJUBETHAPUDI** [comment]
Thank you for your  quick response
The issue solved!!!
